pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
 markers =
     skip_on_windows: mark a test as a test that shouldn't be executed on Windows.
+    sequential: mark tests than cannot be ran in parallel mode.
```

Comment:
what do you think? 
```suggestion
    sequential: mark tests than cannot be ran in parallel (e.g. because they need access to the same resource).
```